### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by bakergo**
I saw a few things which could be cleaned up or extended, feel free to reject some of these (I'm looking at you, no-by-default).

Adds support for 'editor' and 'sensible-editor' (for debian systems).
Defaults to not executing the script
Does some cleanup

### CODE REVIEW

File: shed.py [hunk 1]
```diff
@@ -22,16 +22,19 @@ def default_editor():
     The user's preferred editor. Tries the env vars $SHED_EDITOR and $EDITOR.
     If those variables aren't set, tries nano, vim, vi, and emacs.
```
Comment (mplewis) [inline shed.py]: I should probably clean up that to include sensible-editor and editor too.

File: shed.py [hunk 2]
```diff
@@ -22,16 +22,19 @@ def default_editor():
...
+                        'emacs')
+
+    for var in editor_varibles:
+        if var in os.environ:
             return os.environ[var]
-        except KeyError:
-            pass
 
-    for editor in FALLBACK_EDITORS:
+    for editor in fallback_editors:
```
Comment (mplewis) [inline shed.py:38]: :+1: Caps is whatever

File: shed.py [hunk 3]
```diff
@@ -57,31 +60,29 @@ def open_editor(filename):
...
-    yes = ('yes', 'y')
-    no = ('no', 'n')
-
     print()  # Pretty print spacing
     raw_resp = input('Do you still want to execute this script? [Y/n]: ')
 
     while True:
         resp = raw_resp.strip().lower()
-        if resp in yes:
+        if resp in 'yes':
```
Comment (mplewis) [inline shed.py]: Won't this confirm if the user types just <kbd>e</kbd> or <kbd>s</kbd> too?

File: shed.py [hunk 4]
```diff
@@ -107,18 +108,18 @@ def main():
     os.dup2(tty.fileno(), 0)
 
     # Create a temp file to hold our piped script
-    with NamedTemporaryFile() as f:
-        f.write(script.encode('utf-8'))
-        f.flush()
-        open_editor(f.name)
+    with NamedTemporaryFile() as temp:
```
Comment (mplewis) [inline shed.py:111]: Naming stuff is probably a good idea :smile:

File: shed.py [hunk 5]
```diff
@@ -107,18 +108,18 @@ def main():
...
+        temp.write(script.encode('utf-8'))
+        temp.flush()
+        open_editor(temp.name)
 
         # Ask user for permission to execute modified script
-        if confirm_exec(f.name):
+        if confirm_exec():
```
Comment (mplewis) [inline shed.py:117]: OK, now I see why we don't need the filename arg

File: shed.py (hunk 3 above)
Comment (bakergo) [inline shed.py]: Oops, changed.

File: shed.py (hunk 1 above)
Comment (bakergo) [inline shed.py]: Done.

**Comment by mplewis** [comment]
Looks good to me! Thanks.

**Pull request merged**